examples/specs/normalized/trellis_row_column_normalized.vl.json

```diff
@@ -23,6 +23,14 @@
       "y": {
         "field": "Miles_per_Gallon",
         "type": "quantitative"
+      },
+      "color": {
+        "field": "Cylinders",
+        "type": "nominal"
+      },
+      "shape": {
```

Comment:
I still don't understand how adding these two channel help with testing. 

Also, we will probably need to add one example of trellis with both row and column to the doc and adding these two channels make this example unnecessarily complicated.